src/saga/adaptors/context/myproxy.py

```diff
@@ -21,7 +21,18 @@
 #
 _ADAPTOR_NAME          = 'saga.adaptor.myproxy'
 _ADAPTOR_SCHEMAS       = ['MyProxy']
-_ADAPTOR_OPTIONS       = []
+_ADAPTOR_OPTIONS       = [
+    {
+    'category'         : 'saga.adaptor.myproxy',
+    'name'             : 'base_workdir',
+    'type'             : str,
+    'default'          : "$HOME/.saga/adaptors/proxies/",
+    'documentation'    : '''The adaptor stores job state information on the
+                          filesystem on the target resource.  This parameter
```

Comment:
I think this docstring has been copied, and should probably fixed to mention `proxies` instead of `job state`